package-lock.json

```diff
@@ -0,0 +1,105 @@
+{
```

Comment:
Tried to leave a comment spanning the whole file but couldn't...

After reading this StackOverflow article it's clear to me how the `package-log.json` differs from `yarn.lock`. And it would be, perhaps, nicer for us to have both files in our repository since this is open source, like they suggest. However, we need to, like one user pointed out, have in that case an process that makes sure those files stay in sync and we don't.
I would therefore move a change of this sort to its own branch.
For now please use `yarn` to install your packages and make sure that `yarn.lock` gets updated, and re-run all your tests.

Reply:
Aight, done.